### ISSUE TITLE
implementation with pytorch

### ISSUE THREAD

**Issue by user-832** [Martlgap/octuplet-loss#4]
Dear @Martlgap , 
first of all thank you for this magnificent work you have done. I wanted to ask you the steps I should take to achieve this face recognition with pytorch. I saw that in the github he released a python file that uses pytorch, I think it is the neural network. To implement everything and make it work what steps should I do starting from the 'pt_octuplet_loss.py' file. 
Sorry for disturbing. 
Thank you in advance for your reply.

**Comment by Martlgap** [comment]
pt_octuplet_loss.py is only the loss function. You have to implement the neural network and training yourself.
